packages/flutter/test/material/material_button_test.dart

```diff
@@ -884,4 +884,11 @@ void main() {
     final RawMaterialButton rawMaterialButton = tester.widget(rawMaterialButtonFinder);
     expect(rawMaterialButton.disabledElevation, equals(0.0));
   });
+
+  testWidgets('MaterialButton assertion - negative disabledElevation', (WidgetTester tester) async {
```

Comment:
@shihaohong I think that we should add this test case, just to make sure it doesn't break later.